### PR TITLE
an/SKY-6: Profile detail design

### PR DESCRIPTION
**Pull request by anurmamedov**
1. "edit" button dragged up on top
2. "save changes" button removed
3.  Username Added on top , first name and last name removed
4. Occupation, Bio, No# of posts , liked posts, disliked posts , joined date, last visited dated added.
5. Size of column re-adjusted for each div. 
6. Main div's column set on 11.
7. No clashes on on dimensions: Responsive.
----------------------------------*-----------------------*-------------------
* Comments modified.
* Pictures attached (small, medium, large)

*fontawesome added
*icons added
*more posts added

large profile-details post
medium size profile-details post
small size post images
small size profile-details post

### CODE REVIEW

File: templates/profile-details.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../static/bootstrap/bootstrap.min.css">
+    <link rel="stylesheet" href="../static/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
+    <script src="../static/bootstrap/bootstrap.min.js"></script>
+
+    <title>Profile Details</title>
+</head>
```
Comment (Aymammet) [inline templates/profile-details.html:13]: Why we need these lines? They are already exists in base.html?

File: templates/profile-details.html [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+<body>
+    <div style="background-color: whitesmoke;" class="container">
```
Comment (Aymammet) [inline templates/profile-details.html:15]: use style.css for custom styles

File: templates/profile-details.html [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        <div class="main-body">
+            <nav aria-label="breadcrumb" class="main-breadcrumb text-white">
+                <ol class="breadcrumb text-white">
+                    <li class="breadcrumb-item"><a href="index.html">Home</a></li>
+                    <li class="breadcrumb-item"><a href="javascript:void(0)">Profile</a></li>
```
Comment (Aymammet) [inline templates/profile-details.html:20]: what is javascript : void(0) here?

File: templates/profile-details.html [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+            <div class="row gutters-sm " style="background-color: orangered;">
+                <!-- START OF LEFT PART OF COLUMN -->
+                <div class="col-md-6 col-lg-5 mb-3" style="background-color: green;">
+                    <div class="card">
+                        <div class="card-body">
+                            <div class="d-flex flex-column align-items-center text-center">
+                                <img src="https://bootdey.com/img/Content/avatar/avatar7.png" alt="Admin" class="rounded-circle" width="150">
+                                <div class="mt-3 col-11">
+                                    <h4 class="">@Aykos</h4>
+                                    <h6 class="mb-4 fs-6" style="color:grey">Username</h6>
```
Comment (Aymammet) [inline templates/profile-details.html:33]: Why we need this Username expression here? it is alreade known as username with "@" sign

File: templates/profile-details.html [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <!-- END OF LEFT COLUMN -->
+                </div>
+                <div class="col-md-6 col-lg-7 mb-3" style="background-color: gray;">
+                    POSTS
```
Comment (Aymammet) [inline templates/profile-details.html:132]: Where are posts? This page should be ready with all its components.

File: templates/profile-details.html [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+                </div>
+            </div>
+        </div>
+    </div>    
+</body>
+</html>
```
Comment (Aymammet) [inline templates/profile-details.html:138]:
```suggestion
</html>

```

**Review by Aymammet — CHANGES_REQUESTED** [review]

**Comment by Aymammet** [comment]
Why we have green, gray, orange background colors?

**Comment by Aymammet** [comment]
Why the color of edit button is different from others?

**Comment by Aymammet** [comment]
Sizes of buttons can be same

File: templates/profile-details.html [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+        <div class="main-body">
+            <nav aria-label="breadcrumb" class="main-breadcrumb text-white">
+                <ol class="breadcrumb text-white">
+                    <li class="breadcrumb-item"><a href="">Home</a></li>
+                    <li class="breadcrumb-item"><a href="">Profile</a></li>
+                    <li class="breadcrumb-item active" aria-current="page">User Profile</li>
+                </ol>
+            </nav>
+            <div class="row gutters-sm ">
+                <!-- START OF LEFT PART OF COLUMN -->
```
Comment (Aymammet) [inline templates/profile-details.html:12]: you mena left sidebar?

**Review by Aymammet — CHANGES_REQUESTED** [review]

**Comment by Aymammet** [comment]
Can you send the last images of your changes? large/medium/mobile

**Comment by anurmamedov** [comment]
[profile details updated3] pushed to github.
* Comments modified.
* Pictures attached (small, medium, large)

large size_post detail
medium size_post detail
small size_post detail

**Comment by Aymammet** [comment]
For posts that we display in right side of profile details :

**Comment by Aymammet** [comment]
7. Under the username there are 3 buttons. Their size should be identical for better user experience.
8. What about to use fontawesome icons for some buttons. (as example instead of writing edit we can place fontawesome pen icon)

**Review by Aymammet — CHANGES_REQUESTED** [review]
it is ok for code and design for now. But why we have two same files in template folder. (**profile-detail.html** **and profile-details.html**) ?

**Comment by anurmamedov** [comment]
i made a mistake, renamed it now. back to profile-details.

**Review by Aymammet — APPROVED** [review]

**Pull request merged**